love/src/components/FlightTracker/FlightTracker.jsx

```diff
@@ -0,0 +1,252 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import MapFlightTracker from './MapFlightTracker';
+import Value from '../GeneralPurpose/SummaryPanel/Value';
+import Title from '../GeneralPurpose/SummaryPanel/Title';
+import StatusText from '../GeneralPurpose/StatusText/StatusText';
+import Button from '../GeneralPurpose/Button/Button';
+import isEqual from 'lodash';
+import SimpleTable from 'components/GeneralPurpose/SimpleTable/SimpleTable';
+import ManagerInterface from 'Utils';
+import ZoomInIcon from 'components/icons/Zoom/ZoomInIcon';
+import ZoomOutIcon from 'components/icons/Zoom/ZoomOutIcon';
+import moment from 'moment';
+import { flightTrackerStateToMap, flightTrackerStatetoStyle } from 'Config';
+import styles from './FlightTracker.module.css';
+
+const DEFAULT_POLLING_TIMEOUT = 5000;
+
+export default class FlightTracker extends Component {
+  static propTypes = {
+    status: PropTypes.number,
+    aircrafts: PropTypes.arrayOf(PropTypes.objectOf({
```

Comment:
Please add a Documentation comment for `status` and `aircrafts` params. For the second one I wouldn't define each sub parameter, but a general description, something like:
```javascript
/** Array of all detected aircrafts with their information */
```